core/auth/auth.go

```diff
@@ -139,15 +139,36 @@ func TokenAuth(cfg *config.Configuration) (http.RoundTripper, error) {
 
 // KeyAuth configures the key flow and returns an http.RoundTripper
 // that can be used to make authenticated requests using an access token
+// The KeyFlow requires a service account key and a private key.
```

Comment:
```suggestion
//
// The KeyFlow requires a service account key and a private key.
```
So the comments are shown better formatted